runtime/doc/various.txt

```diff
@@ -19,9 +19,8 @@ CTRL-L			Clear and redraw the screen.  The redraw may happen
 							*:redr* *:redraw*
 :redr[aw][!]		Redraw the screen right now.  When ! is included it is
 			cleared first.
-			Useful to update the screen halfway executing a script
-			or function.  Also when halfway a mapping and
-			'lazyredraw' is set.
+			:redraw can be used to redraw the screen in a script
```

Comment:
Previous text looks better to me

Reply:
What did you prefer about the original text?

Initially I intended just to replace the word "halfway" in the original text.  "...to update the screen _halfway_ executing a script..." doesn't make sense. (Replacing "halfway" with "while" helps: "...to update the screen _while_ executing a script...".)

Reply:
I feel that `:redraw` is redundant as we are right under it already.